p2panda-rs/src/document/document.rs

```diff
@@ -453,7 +455,7 @@ mod tests {
             )
             .unwrap(),
         );
-        let schema = Hash::new_from_bytes(vec![3, 2, 1]).unwrap();
+        let schema = SchemaHash::new(DEFAULT_SCHEMA_HASH).unwrap();
```

Comment:
Grand Type-Tennis :yellow_heart:

Reply:
lolz

Reply:
Seriously! This makes so much sense and the API becomes clearer (like "wtf what hash do I use here for what?")

Reply:
haha, I know, right?? I think we were waiting for some hash typing this whole time :rocket: